GrandCentralBoard/Widgets/Harvest/BillableDates.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  BillableDates.swift
+//  GrandCentralBoard
+//
+//  Created by Maciek Grzybowski on 03.06.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Foundation
+
+
+final class BillableDates {
```

Comment:
Just one more thing I thought of today. Do we need `BillableDates` class, since it is holding only `[NSDate]`? Maybe you could create an extension on `NSDate` instead, that returns an array?